mmv1/third_party/terraform/services/storage/resource_storage_bucket.go.erb

```diff
@@ -760,6 +770,14 @@ func resourceStorageBucketUpdate(d *schema.ResourceData, meta interface{}) error
 		sb.IamConfiguration = expandIamConfiguration(d)
 	}
 
+	if d.HasChange("rpo") {
+		if v,ok := d.GetOk("rpo"); ok{
+			sb.Rpo = v.(string)
+		} else {
+			sb.NullFields = append(sb.NullFields, "Rpo")
```

Comment:
To match API behavior I will say. We are not setting default for this field because of the location conflict so removal of attribute from the user terraform config file will be considered as passing null value by user so to handle this case i have specifically appended Rpo in NullFields. Your suggestions are welcome. Let me know if more clarification needed.

Reply:
SGTM, thanks!